go/sdk/v3/methods.go

```diff
@@ -82,7 +87,7 @@ func NewLookerSDK(session *rtl.AuthSession) *LookerSDK {
 func (l *LookerSDK) Login(request RequestLogin,
     options *rtl.ApiSettings) (AccessToken, error) {
     var result AccessToken
-    err := l.session.Do(&result, "POST", "/login", map[string]interface{}{"client_id": request.ClientId, "client_secret": request.ClientSecret}, nil, options)
+    err := l.session.Do(&result, "POST", "/3.1", "/login", map[string]interface{}{"client_id": request.ClientId, "client_secret": request.ClientSecret}, nil, options)
```

Comment:
For the other RTLs, the ApiVersion goes into the constructor for the base class (e.g. your `LookerSDK`) so you don't need the `"/3.1"` or `"/4.0" proliferation for every method

Reply:
right, yeah, that would be nicer 👍